### DISCUSSION TITLE
Systemd-timesyncd - sync issue

### DISCUSSION THREAD

**Discussion by ammnt** [AdguardTeam/AdGuardHome#2734] [Q&A]
- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * v0.105.0-SNAPSHOT-f2c6e1c6
* **How did you setup DNS configuration:**
  * VM VirtualBox
* **Operating system and version:**
  * Debian 10

### Actual Behavior
I configured time synchronization using Systemd-timesyncd. Each time the AGH server starts, it uses the wrong time (+3 hours). After a couple of minutes (after synchronization), he begins to use the correct time. It's not critical to me, but it's a bit annoying. Maybe I was mistate in configuration somewhere...😤 Please, advice!😟

### Screenshots

<details><summary>Screenshot:</summary>

![2021-01-15_09-27-15 (2)](https://user-images.githubusercontent.com/45385632/104690094-d8037600-5714-11eb-89c7-176e9ed26ff4.png)
![2021-01-15_09-28-05](https://user-images.githubusercontent.com/45385632/104690098-d934a300-5714-11eb-82e8-0eae95e1ee96.png)
![2021-01-15_09-29-35](https://user-images.githubusercontent.com/45385632/104690100-d934a300-5714-11eb-93cd-c357a6c1be93.png)

</details>

### Additional Information
The [detailed logs](https://github.com/AdguardTeam/AdGuardHome/files/5818846/log.txt) is attached.

**Comment by ameshkov** [comment]
Tbh I have no idea, maybe someone else lurking here will see this question and can help you
